overlay.go

```diff
@@ -568,8 +546,8 @@ func (o *Overlay) CreateProtocolService(name string, t *Tree, sid ServiceID) (Pr
 }
 
 // StartProtocol will create and start a P.I.
-func (o *Overlay) StartProtocol(t *Tree, name string) (ProtocolInstance, error) {
-	pi, err := o.CreateProtocolOnet(name, t)
+func (o *Overlay) StartProtocol(name string, t *Tree, sid ServiceID) (ProtocolInstance, error) {
+	pi, err := o.CreateProtocol(name, t, sid)
```

Comment:
+1 for unification in arguments.
Please add some more comment and complete "P.I." - a casual reader might not now what "P.I." is.